src/main/java/com/nashtech/controller/VehicleController.java

```diff
@@ -0,0 +1,54 @@
+package com.nashtech.controller;
+
+import com.nashtech.service.VehicleService;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import reactor.core.publisher.Flux;
+import java.time.Duration;
+
+/**
+ * This controller class handles the endpoints related to
+ * vehicle information retrieval.
+ */
+@RestController
+public class VehicleController {
```

Comment:
add a GCP prefix in the controller name, so our GCP separation should be clear.

Reply:
okay i will update the name